packages/xo-web/src/common/intl/messages.js

```diff
@@ -597,11 +597,14 @@ const messages = {
   vmsTabName: 'Vms',
   srsTabName: 'Srs',
   // ----- Pool advanced tab -----
-  poolHaStatus: 'High Availability',
-  poolHaEnabled: 'Enabled',
+  poolEditRemoteSyslog: 'Edit remote syslog for all hosts',
+  poolGpuGroups: 'GPU groups',
   poolHaDisabled: 'Disabled',
+  poolHaEnabled: 'Enabled',
+  poolHaStatus: 'High Availability',
```

Comment:
Why these changes?

Reply:
sort :)

Reply:
Ok but the messages aren't sorted anyway so let's keep the diff as small as possible.